[2024-03-29T17:01:54.019+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_templating.t6 manual__2024-03-29T17:01:50.584301+00:00 [queued]>
[2024-03-29T17:01:54.130+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_templating.t6 manual__2024-03-29T17:01:50.584301+00:00 [queued]>
[2024-03-29T17:01:54.133+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-29T17:01:54.297+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): t6> on 2024-03-29 17:01:50.584301+00:00
[2024-03-29T17:01:54.357+0000] {standard_task_runner.py:60} INFO - Started process 3245 to run task
[2024-03-29T17:01:54.413+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'demo_templating', 't6', 'manual__2024-03-29T17:01:50.584301+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/demo_templating.py', '--cfg-path', '/tmp/tmp6x916xfm']
[2024-03-29T17:01:54.502+0000] {standard_task_runner.py:88} INFO - Job 49: Subtask t6
[2024-03-29T17:01:54.837+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-29T17:01:55.404+0000] {task_command.py:423} INFO - Running <TaskInstance: demo_templating.t6 manual__2024-03-29T17:01:50.584301+00:00 [running]> on host c4cb237119ff
[2024-03-29T17:01:56.819+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='domwatcharin' AIRFLOW_CTX_DAG_ID='demo_templating' AIRFLOW_CTX_TASK_ID='t6' AIRFLOW_CTX_EXECUTION_DATE='2024-03-29T17:01:50.584301+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-29T17:01:50.584301+00:00'
[2024-03-29T17:01:56.857+0000] {demo_templating.py:10} INFO - Feb 01, 2022
[2024-03-29T17:01:56.860+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-29T17:01:57.008+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=demo_templating, task_id=t6, execution_date=20240329T170150, start_date=20240329T170154, end_date=20240329T170157
[2024-03-29T17:01:57.944+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-29T17:02:00.868+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
